openupgrade_scripts/scripts/account_check_printing/14.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,39 @@
+from openupgradelib import openupgrade
+
+
+def add_preferred_payment_method_id_field_account_move(env):
+    if not openupgrade.column_exists(env.cr, "account_move", "preferred_payment_method_id"):
+        openupgrade.logged_query(
+            env.cr,
+            """
+            ALTER TABLE account_move
+            ADD COLUMN preferred_payment_method_id integer
+            """,
+        )
+        openupgrade.logged_query(
+            env.cr,
+            """
+            UPDATE account_move am
+            SET preferred_payment_method_id = split_part(ip.value, ',', 2);
+            FORM ir_property ip
```

Comment:
there is a typo here. ``FORM`` -> ``FROM``.

it makes travis failing.

https://github.com/OCA/OpenUpgrade/runs/4723892949?check_suite_focus=true#step:13:2428